python/python/bystro/ancestry/train_utils.py

```diff
@@ -30,3 +30,30 @@ def get_variant_ids_from_callset(callset: dict[str, Any]) -> np.ndarray:
 def head(xs: Iterable[T]) -> T:
     """Get first element of xs."""
     return next(iter(xs))
+
+
+VARIANT_REGEX = re.compile(
+    r"""
+    ^
+    chr(1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22)  # (autosomal) chromosome
+    :
+    [0-9]+  # position
+    :
+    [ACGT]  # ref allele
+    :
+    [ACGT]  # alt allele
+    $
+    """,
+    re.VERBOSE,
+)
+
+
+def is_autosomal_variant(potential_variant: str) -> bool:
```

Comment:
Do you want to add a more explicit check whether the chr isn't X,Y,MT here and then you could add this into the liftover step instead of having that more spread out across the loading snp chip and liftover steps? Will mark in define_callset

Reply:
yeah you've def picked up on one of the fault lines in this design.  Conceptually, I think we actually want to make two distinct checks: 

1.  Is it a variant we want to keep?  (i.e. an autosomal variant)
2. Is it a variant that would break liftover?  (an MT variant)

The reason we want to keep these checks distinct is that liftover will fail for a small fraction (perhaps 0.05%) of variants just because they can't be mapped to a unique location in the 38 build-- we monitor the proportion of variants that fail liftover and report it to the user as a sanity check.  Liftover can handle X and Y variants just fine, we simply don't want them in our model.  So if we report X, Y variants as failing in liftover, we'll artificially inflate the liftover failure rate to something that looks alarmingly high, when it doesn't actually signal a problem with liftover.  

But as it is, is_autosomal_variant no longer needs to handle MT variants, so we can take the check out in the interest of simplicity.